router/core/graphql_prehandler.go

```diff
@@ -242,10 +249,17 @@ func (h *PreHandler) Handler(next http.Handler) http.Handler {
 
 				writeOperationError(r, w, requestLogger, err)
 				h.releaseBodyReadBuffer(buf)
+				operationReadSpan.End()
```

Comment:
Attributes aren't added once the span has ended. We need to preserve as much data as possible.

Reply:
We're returning after this (error case). What would you do differently?